package.json

```diff
@@ -23,13 +23,23 @@
   },
   "dependencies": {
     "@module-federation/utilities": "^1.7.0",
+    "@patternfly/patternfly": "^5.4.2",
     "@theforeman/vendor": "^15.0.0",
+    "bootstrap-sass": "^3.4.3",
+    "datatables.net-bs": "1.13.5",
+    "dsmorse-gridster": "^0.8.0",
     "graphql-tag": "^2.11.0",
     "intl": "~1.2.5",
     "jed": "^1.1.1",
     "js-cookie": "^3.0.5",
+    "multiselect": "~0.9.12",
     "os-browserify": "^0.3.0",
-    "react-intl": "^2.8.0"
+    "patternfly-react-extensions": "^3.0.15",
+    "patternfly-react": "^2.40.0",
+    "patternfly": "^3.59.5",
+    "react-diff-view": "^2.6.0",
+    "react-intl": "^2.8.0",
+    "select2": "4.0.12"
```

Comment:
same question, any reason to version-lock this? there is at least a 4.0.13 (and newer non .z)